secator/celery.py

```
@@ -200,6 +200,81 @@ def start_runner(self, config, targets, results=[], run_opts={}, hooks={}, valid
+	Returns:
+		int: Number of times this task has been delivered, or 0 if unsupported.
+	"""
+	backend = app.backend
+	key = backend.get_key_for_task(f'worker-loss-{task_id}')
+
+	# Preferred: atomic incr (Redis, Memcached) — race-free.
+	try:
+		return backend.incr(key)
+	except NotImplementedError:
+		pass  # Backend has no atomic counter; fall back to get/set below.
+	except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)